File: PASCAL/Cap05/c05ex02.pas
program c05ex02;

var
  RESP: String;
  N, R: Integer;

begin
  RESP := 'S';
  while (UpCase(RESP) = 'S') do
    begin
      Write('Entre um valor numerico: ');
      ReadLn(N);

      R := N * 3;

      WriteLn('Resultado = ', R);
      WriteLn;

      Write('Deseja continuar? (S/N): ');
      ReadLn(RESP);
      WriteLn;
    end;

  WriteLn;
  Write('Tecle <Enter> para encerrar... ');
  ReadLn;
end.

File: LUA/Cap05/c05ex02.lua
-- c05ex02.lua

resp = "S"
while (resp:upper() == "S") do
  io.write("Entre um valor numerico: ")
  n = tonumber(io.read())

  r = n * 3

  print("Resultado = " .. r)
  print()

  io.write("Deseja continuar? (S/N): ")
  resp = io.read():sub(1, 1):upper()

  print()
end

print()
io.write("Tecle <Enter> para encerrar... ")
io.read()
